### PR TITLE
Update User Mailer

### PR DESCRIPTION
**Pull request by JasonVinomofo**

### CODE REVIEW

File: app/mailers/user_mailer.rb [hunk 1]
```diff
@@ -4,4 +4,10 @@ def registration_confirmation(user)
     @url  = confirm_url(@user.confirmation_token)
     mail(to: @user.email, subject: 'Welcome to the MofoBottleO')
   end
+
+  def change_password(user)
+    @user = user
+    @url  = confirm_url(@user.confirmation_token)
```
Comment (michaeljd) [inline app/mailers/user_mailer.rb]: What is `@url` used for??

Comment (JasonVinomofo) [inline app/mailers/user_mailer.rb]: so the user can 'copy and paiste' if they wanted and for the text mailer

Comment (michaeljd) [inline app/mailers/user_mailer.rb]: Nope

File: app/views/user_mailer/change_password.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <h1><%= t('.opening', email: @user.email) %></h1>
+
+    <p><%= t('.instructions') %></p>
+
+    <p>
+      <%= link_to t('.link_text', default: 'Change my password'),
+        edit_user_password_url(@user, token: @user.confirmation_token.html_safe) %>
+    </p>
+
+    <p><%= raw t('.closing') %></p>
```
Comment (michaeljd) [inline app/views/user_mailer/change_password.html.erb]: Where is the rest of this page?

Comment (JasonVinomofo) [inline app/views/user_mailer/change_password.html.erb]: doh!

File: config/locales/clearance.en.yml [hunk 3]
```diff
@@ -1,16 +1,13 @@
 ---
 en:
+  # clearance_mailer:
+  #   change_password:
+  #     closing: "If you didn't request this, ignore this email. Your password has not been changed."
+  #     link_text: Change my password
+  #     opening: "Someone, hopefully you, requested we send you a link to change your password:"
```
Comment (michaeljd) [inline config/locales/clearance.en.yml]: `header` or `title` is probably a better name than `opening`

File: config/locales/clearance.en.yml [hunk 4]
```diff
@@ -1,16 +1,13 @@
 ---
 en:
+  # clearance_mailer:
+  #   change_password:
+  #     closing: "If you didn't request this, ignore this email. Your password has not been changed."
```
Comment (michaeljd) [inline config/locales/clearance.en.yml]: `footer` is probably a better name than `closing`

File: config/locales/clearance.en.yml (hunk 3 above)
Comment (JasonVinomofo) [inline config/locales/clearance.en.yml]: Good point!  I'm on it

**Pull request merged**